package.json

```diff
@@ -52,7 +52,7 @@
     "cors": "^2.8.5",
     "express": "^4.17.2",
     "express-handlebars": "^6.0.3",
-    "jest-environment-jsdom": "^29.0.0",
+    "jest-environment-jsdom": "^28.1.3",
```

Comment:
Can we consider splitting this PR?

For instance, the jest-environment-jsdom can be updated independently and it would help to understand the rational of the downgrade.

Reply:
Seriously, split it to a one-line PR?
The rationale is: I downgraded to have the `jest-environment-jsdom` version match `jest` version, because it triggered warnings in the CI. I _guess_ we could revert this change but I think it's good to have consistency in versions.

Reply:
Ok thank you for the rational.
Yes true, no value of having a one-line PR.